sda/cmd/ingest/ingest.go

```diff
@@ -388,6 +390,20 @@ func main() {
 							continue mainWorkLoop
 						}
 
+						// Set the file's hex encoded public key
+						log.Debugln("Compute and set key hash")
+						publicKey := keys.DerivePublicKey(*key)
+						keyhash := hex.EncodeToString(publicKey[:])
+						err = db.SetKeyHash(keyhash, fileID)
+						if err != nil {
+							log.Errorf("Key hash %s could not be set for fileID %s: (%s)", keyhash, fileID, err.Error())
+							if err = delivered.Nack(false, true); err != nil {
```

Comment:
I would use a different variable name for the inner `err` to increase the clarity. However, maybe this not inline with the style of `sda`? What do others think?

Reply:
The inner `err` is scoped to the `Nack` and not accessible after line 402 so there are no change of misreading the error from line 397 as the error from line 400 or vice versa.

Reply:
ok.